code/modules/shuttle/marine_dropship.dm

```diff
@@ -197,6 +197,23 @@
 				var/obj/machinery/door/airlock/multi_tile/mainship/dropshiprear/D = i
 				D.release()
 
+/obj/docking_port/mobile/marine_dropship/proc/prevent_silicon_intervention()
+	silicon_lock_airlocks("rear")
+	silicon_lock_airlocks("left")
+	silicon_lock_airlocks("right")
+
+/obj/docking_port/mobile/marine_dropship/proc/silicon_lock_airlocks(side)
+	switch(side)
+		if("left")
+			for(var/obj/machinery/door/airlock/dropship_hatch/D AS in left_airlocks)
+				D.aiControlDisabled =! D.aiControlDisabled
+		if("right")
+			for(var/obj/machinery/door/airlock/dropship_hatch/D AS in right_airlocks)
+				D.aiControlDisabled =! D.aiControlDisabled
+		if("rear")
+			for(var/obj/machinery/door/airlock/multi_tile/mainship/dropshiprear/D AS in rear_airlocks)
+				D.aiControlDisabled =! D.aiControlDisabled
```

Comment:
This is not how you should do it. 

First, you don't need a switch since you'll always want to lock/unlock all sides at once.
Second, you don't control what you are doing with = !var

I suggest changing the proc to /obj/docking_port/mobile/marine_dropship/proc/silicon_lock_airlocks(should_lock = TRUE)

D.aiControlDisabled = should_lock